sdk/scheduler/src/main/java/com/mesosphere/sdk/api/PlansResource.java

```diff
@@ -149,6 +151,20 @@ public Response continueCommand(
         return jsonOkResponse(getCommandResult("continue"));
     }
 
+    /**
+     * Interrupts (Pauses) a plan or if specified a phase within a plan.  If a plan/phase is in progress, it
```

Comment:
It may make sense to have this information in the ops guide as well. Doesn't have to be done in this PR, but may be worth filing a ticket at least. Looks like the only mention at the moment is here, where a 'restart' command is issued to unstick a step: https://mesosphere.github.io/dcos-commons/operations-guide.html#stuck-deployments